internal/buildscripts/packaging/msi/splunk-support-bundle.ps1

```diff
@@ -0,0 +1,263 @@
+        New-Item -Path $TMPDIR/zpages/$global:DIRECTORY/debug -ItemType Directory | Out-Null
+    }
+}
+
+#######################################
+# Gather configuration
+# Without this it is very hard to troubleshoot issues so exit if no permissions.
+#  - GLOBALS: CONFDIR, DIRECTORY, TMPDIR
+#  - ARGUMENTS: None
+#  - OUTPUTS: None
+#  - RETURN: 0 if successful, non-zero on error.
+#######################################
+function getConfig {
+    Write-Output "INFO: Getting configuration..."
+    # If directory does not exist the support bundle is useless so exit
+    if (-NOT (Test-Path -Path $CONFDIR)) {
+        Write-Output "ERROR: Could not find directory ($CONFDIR)."
+        usage
+    } else {
+        Copy-Item -Path "$CONFDIR" -Destination "$TMPDIR/config" -Recurse
+    }
+    # Also need to get config in memory as dynamic config may modify stored config
+    # It's possible user has disabled collecting in memory config
+    $connection = New-Object System.Net.Sockets.TcpClient("localhost", 55554)
+    if ($connection.Connected) {
+        (Invoke-WebRequest -Uri "http://localhost:55554/debug/configz/initial").Content > $TMPDIR/config/initial.yaml 2>&1
+        (Invoke-WebRequest -Uri "http://localhost:55554/debug/configz/effective").Content > $TMPDIR/config/effective.yaml 2>&1
+    } else { 
+        Write-Output "WARN: localhost:55554 unavailable so in memory configuration not collected"
+    }
+}
+
+#######################################
+# Gather status
+#  - GLOBALS: TMPDIR
+#  - ARGUMENTS: None
+#  - OUTPUTS: None
+#  - RETURN: 0
+#######################################
+function getStatus {
+    Write-Output "INFO: Getting status..."
+    Get-Service splunk-otel-collector -ErrorAction SilentlyContinue > $TMPDIR/logs/splunk-otel-collector.txt 2>&1
+    Get-Service fluentdwinsvc -ErrorAction SilentlyContinue > $TMPDIR/logs/td-agent.txt 2>&1
+    if (-NOT (Get-Content -Path "$TMPDIR/logs/splunk-otel-collector.txt")) {
+        Set-Content -Path "$TMPDIR/logs/splunk-otel-collector.txt" -Value "Service splunk-otel-collector not exist.."
+        Write-Output "WARN: Service splunk-otel-collector not exist.."
+    }
+    if (-NOT (Get-Content -Path "$TMPDIR/logs/td-agent.txt")) {
+        Set-Content -Path "$TMPDIR/logs/td-agent.txt" -Value "Service td-agent not exist.."
+        Write-Output "WARN: Service td-agent not exist.."
+    }
+}
+
+#######################################
+# Gather logs
+#  - GLOBALS: TMPDIR
+#  - ARGUMENTS: None
+#  - OUTPUTS: None
+#  - RETURN: 0
+#######################################
+function getLogs {
+    Write-Output "INFO: Getting logs..."
+    Get-EventLog -LogName Application -Source "splunk-otel-collector" -ErrorAction SilentlyContinue > $TMPDIR/logs/splunk-otel-collector.log 2>&1
+    Get-EventLog -LogName Application -Source "td-agent" -ErrorAction SilentlyContinue > $TMPDIR/logs/td-agent.log 2>&1
+    $LOGDIR="/var/log/td-agent"
```

Comment:
Hi @jcheng-splunk 
while testing we have installed td-agent using MSI installation and we found that td-agent system log directory is by default set to `c:/var/log/td-agent` so can I make it `${env:SYSTEMDRIVE}\var\opt\td-agent`?

Reply:
Maybe it is both?  I do see `${env:SYSTEMDRIVE}\opt\td-agent\td-agent.log`.

Reply:
Okay, so it may differ from installation ways so we can keep both.

I have made all the suggested changes, let us know if more changes are required.